drivers/rptun/rptun.c

```diff
@@ -56,9 +56,10 @@
 #define RPTUNIOC_NONE               0
 #define NO_HOLDER                   (INVALID_PROCESS_ID)
 
-#define RPTUN_STATUS_FROM_MASTER    0x8
-#define RPTUN_STATUS_MASK           0x7
-#define RPTUN_STATUS_PANIC          0x7
+#define RPTUN_OPS_START             0
```

Comment:
let's reuse the ioctl id here?

Reply:
@GUIDINGLI please issue a new PR if needed to address this

Reply:
@GUIDINGLI please issue a new PR if needed to address this